tests/helpers/participants.ts

```diff
@@ -78,3 +163,64 @@ export async function toggleMuteAndCheck(testee: Participant, observer: Particip
     await observer.getFilmstrip().assertAudioMuteIconIsDisplayed(testee);
     await testee.getFilmstrip().assertAudioMuteIconIsDisplayed(testee);
 }
+
+/**
+ * Parse a JID string.
+ * @param str the string to parse.
+ */
+export function parseJid(str: string): {
```

Comment:
There was a Strophe helper to do this, wasn't there? Do we want to use it?

Reply:
Yeah, I was thinking the same ... but was wondering about the dependencies ... but the change will be only in dev dependencies ... I will give it a shot.

Reply:
Ah good point. Never mind, keep it as is.